test/kubemark/start-kubemark.sh

```diff
@@ -289,44 +315,73 @@ if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
     fi
   done
 
-  echo "DBG: Starting resource partition ..."
+  echo "DBG: Starting ${SCALEOUT_RP_COUNT} resource partitions ..."
   export KUBERNETES_TENANT_PARTITION=false
   export KUBERNETES_RESOURCE_PARTITION=true
-  create-kubemark-master
+  for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
+  do
```

Comment:
start hollow-nodes should be as part of RP start up to keep them as a single entity.